R/twas.R

```diff
@@ -180,7 +179,7 @@ twas_analysis <- function(weights_matrix, gwas_sumstats_db, LD_matrix, extract_v
 #' @importFrom furrr future_map furrr_options
 #' @importFrom purrr map
 #' @export
-twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, weight_methods = NULL, seed = NULL, max_num_variants = NULL, variants_to_keep = NULL, num_threads = 1, imiss_cutoff = 1.0, min_cv_maf = 0.05, ...) {
+twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, weight_methods = NULL, max_num_variants = NULL, variants_to_keep = NULL, num_threads = 1, imiss_cutoff = 1.0, ...) {
```

Comment:
are you still using `imiss_cutoff` somewhere? I dont think that is the case any more. Don't we do QC at least on imiss_cutoff for each fold of CV? i

Reply:
No, it won't be used anywhere inside of twas cv, it is now removed in the upcoming PR.